src/components/VolunteerAutomation/Stepper.js

```diff
@@ -0,0 +1,258 @@
+import React, { useState } from "react";
+import { PATHS } from "../../constant";
+import Box from "@mui/material/Box";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import { useHistory } from "react-router-dom";
+import { StepLabel } from "@mui/material";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { Container } from "@mui/material";
+import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import SelectTrack from "./SelectTrack";
+import Confirmation from "./Confirmation";
+import AttendClass from "./AttendClass";
+import CodeOfConduct from "./CodeOfConduct";
+import VerifyPhoneNo from "./VerifyPhoneNo";
+import IntroVideo from "./IntroVideo";
+import axios from "axios";
+import { useSelector } from "react-redux";
+import { METHODS } from "../../services/api";
+
+import "./styles.scss";
+import { parse } from "date-fns";
+
+function HorizontalLinearStepper() {
+  let history = useHistory();
+  const myData = localStorage.getItem("step");
+  const user = useSelector(({ User }) => User);
+  const [activeStep, setActiveStep] = React.useState(
+    myData ? parseInt(myData) : 0
+  );
+  const isDisabled = JSON.parse(localStorage.getItem("disabled"));
+  const [skipped, setSkipped] = React.useState(new Set());
+  const [disable, setDisable] = React.useState(
+    isDisabled == false ? isDisabled : true
+  );
+  const [contact, setContact] = useState();
+  const [pathwayId, setPathwayId] = useState();
+
+  const steps = [
+    {
+      label: "Verify Phone No.",
+      component: (
+        <VerifyPhoneNo
+          contact={contact}
+          setContact={setContact}
+          setDisable={setDisable}
+        />
+      ),
+    },
+    {
+      label: "Select Track",
+      component: (
+        <SelectTrack
+          setPathwayId={setPathwayId}
+          pathwayId={pathwayId}
+          setDisable={setDisable}
+        />
+      ),
+    },
+    {
+      label: "Intro Video",
+      component: <IntroVideo setDisable={setDisable} />,
+    },
+    {
+      label: "Code of Conduct",
+      component: <CodeOfConduct setDisable={setDisable} />,
+    },
+    {
+      label: "Attend Class",
+      component: <AttendClass setDisable={setDisable} />,
+    },
+    {
+      label: "Confirmation",
+      component: <Confirmation setDisable={setDisable} />,
+    },
+  ];
+
+  const isStepOptional = (step) => {
+    return step === 1;
```

Comment:
`step 1` isn't optional so since there are no optional steps as of now, you should probably use `return false` here.
```suggestion
    return false;
```